jni/Android-app.mk

```diff
@@ -19,6 +19,7 @@ LOCAL_PATH := $(call my-dir)
 NNSTREAMER_VERSION := 0.1.1
 CUSTOM_LINKER64    := -fPIE -pie -Wl,-dynamic-linker,/data/nnstreamer/libandroid/linker64
 
+NO_AUDIO := false
```

Comment:
Do we have to set-up a default value with "false"?

Reply:
I think it's more readable to set a default value as false even if it automatically assigns false if unspecified.